config/peerpods/aws-VM-image-create-job.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: aws-image-creation
+  namespace: openshift-sandboxed-containers-operator
+spec:
+  parallelism: 1
+  completions: 1
+  backoffLimit: 1
+  template:
+    metadata:
+      name: aws-image-creation
+    spec:
+      volumes:
+      - name: shared-data
+        emptyDir: {}
+        volumes:
+      - name: image-id
+        emptyDir: {}
+
+      securityContext:
+        runAsUser: 0 # needed for container mode dnf access
+
+      initContainers:
+      - name: payload
+        image: registry.redhat.io/openshift-sandboxed-containers/osc-podvm-payload-rhel9:1.5.0
+        imagePullPolicy: Always
+        volumeMounts:
+        - name: shared-data
+          mountPath: /payload
+        command: ["/bin/sh"]
+        args: ["-c", "cp /podvm-binaries.tar.gz /payload/"]
+
+      restartPolicy: Never
+      containers:
+      - name: create
+        image: registry.access.redhat.com/ubi9/ubi:9.2
+        volumeMounts:
+        - name: shared-data
+          mountPath: /payload
+        - name: image-id
+          mountPath: /output
+        env:
+          - name: PODVM_DISTRO
+            value: rhel
+          - name: IMAGE_NAME
+            value: "peer-pod-ami"
+#          - name: INSTANCE_TYPE
+#            value: "t3.small" # default is t3.small, uncomment and modify if not available in your region
+        envFrom:
+        - secretRef:
+            name: peer-pods-secret
+        - configMapRef:
+            name: peer-pods-cm
+            optional: true
+        command:
+        - /bin/sh
+        - -c
+        - |
+          set -e
+          dnf install -y make git unzip
+          curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+          unzip awscliv2.zip
+          ./aws/install
+          curl https://releases.hashicorp.com/packer/1.9.4/packer_1.9.4_linux_amd64.zip -o packer_1.9.4_linux_amd64.zip
+          echo "6cd5269c4245aa8c99e551d1b862460d63fe711c58bec618fade25f8492e80d9  packer_1.9.4_linux_amd64.zip" | sha256sum -c
+          unzip packer_1.9.4_linux_amd64.zip -d /usr/bin/
+          PATH="/usr/bin:${PATH}"
+          packer plugins install github.com/hashicorp/amazon
+          git clone https://github.com/confidential-containers/cloud-api-adaptor.git -b v0.8.0
```

Comment:
This should be from 0.7.0+cherry-picks right ? 0.8.0 is not yet there and also will not match with the CAA and the binaries

Reply:
ah right, how should i tag it